src/minimega/main.go

```diff
@@ -50,6 +50,7 @@ var (
 	f_cli        = flag.Bool("cli", false, "validate and print the minimega cli, in JSON, to stdout and exit")
 	f_panic      = flag.Bool("panic", false, "panic on quit, producing stack traces for debugging")
 	f_cgroup     = flag.String("cgroup", "/sys/fs/cgroup", "path to cgroup mount")
+	f_future     = flag.Bool("test-the-future", false, "disable features that will be removed in future releases, such as targeting VMs by ID")
```

Comment:
Hm, I was thinking the opposite direction for this flag -- log a warning if they target VMs by ID and don't specify `-i-wont-index-vms-by-id-in-the-future` (or something similar).